music-player/src/components/SongsList.tsx

```diff
@@ -0,0 +1,72 @@
+import { List, Button } from 'antd';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+
+import { songsNamesList } from '../data';
+import { State } from '../types';
+import { AddSongToPlaylist } from './AddSongToPlaylist';
+import { showDeleteConfirm } from './RemoveSongFromPlaylist';
+import { getPlaylists, getCurrentPlaylistName, getCurrentSongIndex } from '../reducers';
+
+const SongsList = ({
+  songsToDisplay,
+  currentSongIndex,
+  dispatch,
+  songsIndices,
+  currentPlaylistName,
```

Comment:
This prop is only used to determine if deletion is allowed.
You should just pass in `mapStateToProps`:
```
return {
    ...
    allowDelete: currentPlaylistName === 'All',
    ...
}
```